system-test/integration_test.go

```diff
@@ -289,7 +289,7 @@ func TestAgentIntegration(t *testing.T) {
 			},
 			name:          fmt.Sprintf("profiler-test-node13-%s-gce", runID),
```

Comment:
name says node 13 but  nodeVersion is set to 12

Reply:
Done.